### PR TITLE
setup UITest Cloud for UI testing in Sensus

### PR DESCRIPTION
**Pull request by cadancai**

### CODE REVIEW

File: Sensus.Android/Sensus.Android.csproj [hunk 1]
```diff
@@ -90,27 +90,6 @@
     <Reference Include="Xam.Plugin.MapExtend.Android">
```
Comment (MatthewGerber) [inline Sensus.Android/Sensus.Android.csproj]: Don't include changes to this file.

File: Sensus.UITests/Tests.cs [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    {
+        IApp app;
+        Platform platform;
+
+        /// <summary>
+        /// this is our xamarin test cloud APIkey.
+        /// It could be found in the xamarin test cloud account.
+        /// Each team will have a different APIkey. The one below is from the sensus team.
+        /// </summary>
+        const string _APIkey = "b38984cea752e9d489573615fb5fddaf";
```
Comment (MatthewGerber) [inline Sensus.UITests/Tests.cs]: You should check whether including this API key in the repository is a security risk.

File: Sensus.iOS/Sensus.iOS.csproj [hunk 3]
```diff
@@ -127,9 +127,6 @@
       <HintPath>..\Components\azure-mobile-services-1.3.1\lib\ios-unified\Microsoft.WindowsAzure.Mobile.dll</HintPath>
```
Comment (MatthewGerber) [inline Sensus.iOS/Sensus.iOS.csproj]: Don't include these changes.

**Comment by MatthewGerber** [comment]
Also - don't include the APK. It should be built when the tests are run.

File: Sensus.iOS/Sensus.iOS.csproj (hunk 3 above)
Comment (cadancai) [inline Sensus.iOS/Sensus.iOS.csproj]: These seem to be generated by the system under build.

**Pull request closed without merging**